tests/features/user/password.feature

```diff
@@ -0,0 +1,29 @@
+@api @javascript
+Feature: Password management
+  A user must be able to change his password
+
+  Scenario: A logged-in user can navigate to his profile and change his password.
+    Given users:
+      | name           | mail                        | pass        | First name | Family name |
+      | Charlie Change | charlie.change@example.com  | changeme    | Charlie    | Change      |
```

Comment:
This does not work. The "@Given users" does not take arguments using labels. The names of the field must be machine names.